frontend/src/actions/userActions.js

```diff
@@ -140,7 +156,7 @@ export const followUser = (username) => async (dispatch, getState) => {
 
 export const updateUserProfile = (userData) => async (dispatch, getState) => {
   try {
-    // dispatch({ type: UPDATE_USER_REQUEST });
+    dispatch({ type: LOAD_MORE_USER_REQUEST });
```

Comment:
UPDATE_USER_REQUEST & LOAD_MORE_USER_REQUEST  basically do something,  since I'm using this chunk for loading more users, it looks more appropriate [action type].